projects/ngx-d3timeline/src/lib/event.service.ts

```diff
@@ -0,0 +1,13 @@
+import { BehaviorSubject } from 'rxjs';
+import { Injectable } from '@angular/core';
+
+@Injectable({ providedIn: 'root' })
+export class EventService {
```

Comment:
Is this only (ever) used for zoom events? If so could be named more appropriately. 'EventService' is very generic.